compile error OSRM > 5.1.0

In the past I have succesfully used OSRM 5.1.0 on LINUX Debian GNU/Linux 8 (jessie)
While compiling OSRM > 5.1.0 

`-- Building on a 64 bit system
-- Configuring OSRM in release mode
-- LTO working
-- Using gcc specific binutils for LTO.
-- Setting linker optimizations
-- Boost version: 1.55.0
-- Found the following Boost libraries:
--   date_time
--   filesystem
--   iostreams
--   program_options
--   regex
--   system
--   thread
-- Found Intel TBB
-- TBB interface version: 7003
-- Looking for Luabind...
-- Found Luabind: /usr/lib/libluabind.so
-- Looking for LuaJIT 5.2
-- Could NOT find LUAJIT (missing:  LUAJIT_LIBRARIES LUAJIT_INCLUDE_DIR)
-- Performing Test LUABIND_WORKS
-- Performing Test LUABIND_WORKS - Success
-- Luabind/Lua5.2 combination working with /usr/lib/x86_64-linux-gnu/liblua5.2.so
-- Looking for STXXL...
-- Found STXXL: /usr/lib/libstxxl.so
-- OpenMP support found. Linking just in case for stxxl
-- Enabling json logging
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Boost version: 1.55.0
-- Found the following Boost libraries:
--   unit_test_framework
-- Configuring done
-- Generating done`

I have always the same error when I run `cmake --build .`

`Scanning dependencies of target osrm-contract
[ 95%] Building CXX object CMakeFiles/osrm-contract.dir/src/tools/contract.cpp.o
Linking CXX executable osrm-contract
'_ZN3tbb10empty_taskD1Ev' referenced in section '.data.rel.ro._ZTVN3tbb10empty_taskE[_ZTVN3tbb10empty_taskE]' of /tmp/ccBbchYm.ltrans10.ltrans.o: defined in discarded section '.gnu.linkonce.t._ZN3tbb10empty_taskD5Ev' of contractor.cpp.o (symbol from plugin)
'_ZN3tbb10empty_taskD0Ev' referenced in section '.data.rel.ro._ZTVN3tbb10empty_taskE[_ZTVN3tbb10empty_taskE]' of /tmp/ccBbchYm.ltrans10.ltrans.o: defined in discarded section '.gnu.linkonce.t._ZN3tbb10empty_taskD5Ev' of contractor.cpp.o (symbol from plugin)
collect2: error: ld returned 1 exit status
CMakeFiles/osrm-contract.dir/build.make:99: recipe for target 'osrm-contract' failed
make[2]: *** [osrm-contract] Error 1
CMakeFiles/Makefile2:342: recipe for target 'CMakeFiles/osrm-contract.dir/all' failed
make[1]: *** [CMakeFiles/osrm-contract.dir/all] Error 2
Makefile:117: recipe for target 'all' failed
make: *** [all] Error 2`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -320,7 +320,7 @@ include_directories(SYSTEM ${OSRM_INCLUDE_PATHS})
 # Binaries
 target_link_libraries(osrm-datastore osrm_store ${Boost_LIBRARIES})
 target_link_libraries(osrm-extract osrm_extract ${Boost_LIBRARIES})
-target_link_libraries(osrm-contract osrm_contract ${Boost_LIBRARIES})
+target_link_libraries(osrm-contract ${Boost_LIBRARIES} ${TBB_LIBRARIES} osrm_contract)
 target_link_libraries(osrm-routed osrm ${Boost_LIBRARIES} ${OPTIONAL_SOCKET_LIBS} ${ZLIB_LIBRARY})
 
 set(EXTRACTOR_LIBRARIES

